src/routes/products.ts

```diff
@@ -3,7 +3,11 @@ import * as productsController from "../controllers/productsController";
 import multer from "multer";
 
 const storage = multer.memoryStorage();
-const upload = multer({ storage: storage });
+const upload = multer({ 
+    storage: storage,
+    limits: {
+        fileSize:2000000 // 2Mb 
```

Comment:
This could be written as `fileSize: 2 * 1000 * 1000 // 2Mb` which is more common and helps people understand how you came up with this number.

Btw, 2MB is `2 * 1024 * 1024`